### DISCUSSION TITLE
Enable support for AutoGluon

### DISCUSSION THREAD

**Discussion by bvrabete** [SalesforceAIResearch/uni2ts#65] [Ideas]
[AutoGluon](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-quick-start.html) is an AutoML framework that also offers support for time series forecasting. Most if not all traditional models are supported (Arima, DeepAR) but also new models such as PatchTST and Chronos.

The useful part about that is that it makes fine tuning and comparison versy easy.

Would it b possible to enable AutoGluon support for your model(s)? Or point out how a wrapper class for Moirai could be written?

**Comment by gorold** [comment]
Not familiar with AutoGluon, so can't provide any help on how to write a wrapper for it, but the starting point would be looking at the MoiraiForecast class: https://github.com/SalesforceAIResearch/uni2ts/blob/main/src/uni2ts/model/moirai/forecast.py
